src/Microsoft.ML.Data/DataView/RowToRowMapperTransform.cs

```diff
@@ -134,15 +134,15 @@ public sealed class RowToRowMapperTransform : RowToRowTransformBase, IRowToRowMa
     {
         private sealed class Bindings : ColumnBindingsBase
         {
-            private readonly RowToRowMapperTransform _parent;
+            private readonly IRowMapper _mapper;
```

Comment:
>IRowMapper [](start = 29, length = 10)

Would this stop double mapper creation which we have right now for each RowToRowMapperTransform instantiation?

Reply:
I think yes

---
In reply to: [217230609](https://github.com/dotnet/machinelearning/pull/896#discussion_r217230609) [](ancestors = 217230609)